### ISSUE TITLE
Segmentation fault with hunt 1.4.16

### ISSUE THREAD

**Issue by nu8** [huntlabs/hunt-http#17]
This file:

~~~d
import hunt.http;
import std.stdio;
void main() {
   auto client = new HttpClient();
   auto s1 = "http://speedtest.lax.hivelocity.net";
   auto request = new RequestBuilder().url(s1).build();
   auto response = client.newCall(request).execute();
   if (response !is null) {
      writeln(response.getBody().asString());
   }
}
~~~

Result:

~~~
$ dub add hunt-http
Adding dependency hunt-http ~>0.6.0

$ dub add hunt@1.4.16
Adding dependency hunt 1.4.16

$ dub add hunt
Adding dependency hunt ~>1.6.0

$ dub build
Performing "debug" build using C:\ldc2-1.22.0-beta1-windows-x64\bin\ldc2.exe
for x86_64.
hunt 1.6.0: target for configuration "library" is up to date.
hunt-extra 1.0.0: target for configuration "library" is up to date.
hunt-net 0.5.0: target for configuration "default" is up to date.
hunt-http 0.6.0: target for configuration "default" is up to date.
etc ~master: building configuration "application"...
Linking...
To force a rebuild of up-to-date targets, run again with --force.

$ ./etc.exe
Segmentation fault
~~~

**Comment by muv1** [comment]
Im not sure Dub allows 2 versions of the same package. I noticed the 3rd command just overrides the 2nd. If you reverse, you get this:

~~~
$ dub build
Unresolvable dependencies to package hunt:
  etc ~master depends on hunt 1.4.16
  hunt-extra 1.0.0 depends on hunt ~>1.6.0
~~~

**Comment by Heromyth** [comment]
It's fixed in Hunt 1.6.1 which is released now. Run `dub upgrade`, please.

**Comment by nu8** [comment]
It works, thank you
